### PR TITLE
Refactor: Removed Unneccessary style

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Refactored: Unneccessary style

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

Your vertical cards have different heights. See if you can give same height to all the cards irrespective of the content and images used.

**Review by anik31 — APPROVED** [review]
Everything is good. Just a suggestion to use more descriptive class names.

File: quiz-question/theoffice-quiz.css [hunk 1]
```diff
@@ -5,24 +5,22 @@
...
     background: var(--primary-clr);
-    height: 4rem;
-    padding: 1rem;
     border-radius: 5px;
     gap :1rem;
     justify-content: space-around;
 }
 
 .question-txt{
+    padding-inline: 1rem;
```
Comment (MayankKumar10) [inline quiz-question/theoffice-quiz.css:23]: make a helper class for padding and margin

**Review by MayankKumar10 — COMMENTED** [review]
Look into these, Everything else looks good to me you can merge it

**Pull request merged**